src/Sylius/Behat/Context/Cli/CancelUnpaidOrdersContext.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Cli;
+
+use Sylius\Component\Order\Model\OrderInterface;
+use Sylius\Component\Core\Repository\OrderRepositoryInterface;
+use Webmozart\Assert\Assert;
+use Behat\Behat\Context\Context;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+
+final class CancelUnpaidOrdersContext implements Context
+{
+    private const CANCEL_UNPAID_ORDERS_COMMAND = 'sylius:cancel-unpaid-orders';
+
+    private Application $application;
+
+    private ?CommandTester $commandTester = null;
+
+    private OrderRepositoryInterface $orderRepository;
+
+    public function __construct(KernelInterface $kernel, OrderRepositoryInterface $orderRepository)
+    {
+        $this->application = new Application($kernel);
+        $this->orderRepository = $orderRepository;
+    }
+
+    /**
+     * @When I run cancel unpaid orders command
+     */
+    public function runCancelUnpaidOrdersCommand(): void
+    {
+        $command = $this->application->find(self::CANCEL_UNPAID_ORDERS_COMMAND);
+
+        $this->commandTester = new CommandTester($command);
+        $this->commandTester->execute(['command' => self::CANCEL_UNPAID_ORDERS_COMMAND]);
+    }
+
+    /**
+     * @Then only the order with number :orderNumber should be canceled
+     */
+    public function onlyOrderWithNumberShouldBeCanceled(string $orderNumber): void
+    {
+        $order = $this->orderRepository->findOneByNumber($orderNumber);
+
+        Assert::notEmpty($order);
+        Assert::same($order->getState(), OrderInterface::STATE_CANCELLED);
+    }
```

Comment:
Here it stands "**only** the order with number (...)" but there is no other order checked. I believe we should just iterate over all orders and check if only the one with the given number is cancelled. This way we're not sure if others are not accidentally cancelled as well

Reply:
So we should probably change the name of the step because now the content of the method correspond with the step definition.